src/components/popup/details/details.jsx

```diff
@@ -45,30 +52,27 @@ export default class Details extends Component {
 	};
 
 	handleShowVendors = (filter) => {
-		this.state.vendors = this.getVendors(filter);
+		this.props.store.subsection = SECTION_VENDORS;
```

Comment:
Here we can use `this.props.store.updateSection(SECTION_DETAILS, SECTION_VENDORS);`